components/taggroup/package.json

```diff
@@ -19,16 +19,16 @@
   },
   "peerDependencies": {
     "@spectrum-css/clearbutton": ">=1",
-    "@spectrum-css/tag": ">=3 <=5",
-    "@spectrum-css/vars": ">=9"
+    "@spectrum-css/tag": ">=5",
+    "@spectrum-css/tokens": ">=10"
```

Comment:
![11](https://github.com/adobe/spectrum-css/assets/965114/aad2bc68-fbb9-4677-a13f-b8ab4bad8ebe)

Reply:
I used the minimum `@spectrum-css/tokens` version currently specified in tag, since there are no other newly added tokens needed for tag group. I don't think we need to bump up to 11 yet, unless tag needs to bump up its minimum?

Reply:
Since you were using 11 in your devDependencies I thought it made sense to use it as the baseline but you're probably right that you don't _need_ to yet. Makes sense to leave it to match tag for now!